README.md

```diff
@@ -297,7 +299,24 @@ either enable "debug" dracut module or add `dracut_install netstat ip` line to
 (for "rdsosreport.sh") show - there can be no default route, whatever interface
 naming mixup, no traffic (e.g. unrelated connection issue), etc.
 
-## 5.4 Report a bug
+## 5.4 *dracut: dropbearconvert for ED25519 key failed*
+
+In the case that your dropbearconvert does not support ED25519 encryption (e.g CentOS Stream), you will need to install
+dropbear from source upstream of [this PR](https://github.com/mkj/dropbear/pull/91).  
+
+1. Clone [dropbear git repo](https://github.com/mkj/dropbear)
+
+2. Intsall from source. 
+   ```
+   ./configure --prefix /usr  # sudo has only /sbin:/bin:/usr/sbin:/usr/bin and default prefix is /usr/local
+   make
+   sudo make install
+   ```
+
+3. Now re-run `dracut --force`
```

Comment:
That said, I have issues with this approach as a whole.

- you're cloning from master instead of using a proper release
- you're overriding files controlled by the file manager, which is not recommended on any distro I know of

Reply:
Fair point, maybe `/usr/local/bin/dracut --force` would work just as well?

Reply:
> Fair point, maybe /usr/local/bin/dracut --force would work just as well?
But the calls to `dropbearconvert` would still reference the `dropbearconvert` in `/usr/bin` not those in `/usr/local/bin`.  

Have a workaround that prepends the PATH environment variable for this single sudo call.

Reply:
> you're cloning from master instead of using a proper release

Added a checkout to the latest dropbear release tag

> you're overriding files controlled by the file manager, which is not recommended on any distro I know of

Added a workaround that prefixes the `/usr/local/bin` to the PATH env var for that sudo command.